src/modules/components/proposal/proposalActions/utils/proposalActionsUtils.ts

```diff
@@ -0,0 +1,19 @@
+import { ProposalActionWithdrawToken } from '../actions/proposalActionWithdrawToken/proposalActionWithdrawToken';
+import type { IProposalAction, IProposalActionWithdrawToken } from '../proposalActionsTypes';
+import { ProposalActionType } from '../proposalActionsTypes/proposalAction';
+
+class ProposalActionsUtils {
+    public getActionComponent(action: IProposalAction) {
+        if (this.isWithdrawTokenAction(action)) {
+            return () => ProposalActionWithdrawToken({ action });
+        }
+
+        return null;
+    }
+
+    public isWithdrawTokenAction(action: Partial<IProposalAction>): action is IProposalActionWithdrawToken {
```

Comment:
Just to follow how we're currently defining functions in utilities
```suggestion
   getActionComponent = (action: IProposalAction) => {
        if (this.isWithdrawTokenAction(action)) {
            return () => ProposalActionWithdrawToken({ action });
        }

        return null;
    }

    isWithdrawTokenAction = (action: Partial<IProposalAction>): action is IProposalActionWithdrawToken => {
```